app/controllers/employees_controller.rb

```diff
@@ -56,6 +63,10 @@ def employee_params
     params.require(:employee).permit(:slack_username, :started_on, :time_zone)
   end
 
+  def existing_employee(slack_username)
+    !!Employee.where(slack_username: slack_username)
```

Comment:
I am not a huge an of `!!` -- I think `Employee.where(slack_username: slack_username).any?` would be easy to read